### PR TITLE
font-noto-nerd-font, font-noto-nerd-font-mono v2.0.0

### PR DESCRIPTION
**Pull request by alindeman**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask-fonts/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask-fonts/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Why include the Windows compatible fonts?

**Comment by alindeman** [comment]
@vitorgalvao Good point, the file was mostly autogenerated, but I've updated the diff to remove the 'Windows Compatible' fonts.

File: Casks/font-noto-nerd-font-mono.rb [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+  font 'Noto Sans Mono ExtraCondensed Nerd Font Complete Mono.ttf'
+  font 'Noto Sans Mono ExtraCondensed Nerd Font Complete.ttf'
+  font 'Noto Sans Mono ExtraCondensed SemiBold Nerd Font Complete Mono.ttf'
+  font 'Noto Sans Mono ExtraCondensed SemiBold Nerd Font Complete.ttf'
+  font 'Noto Sans Mono ExtraCondensed Thin Nerd Font Complete Mono.ttf'
+  font 'Noto Sans Mono ExtraCondensed Thin Nerd Font Complete.ttf'
+  font 'Noto Sans Mono ExtraLight Nerd Font Complete Mono.ttf'
+  font 'Noto Sans Mono ExtraLight Nerd Font Complete.ttf'
+  font 'Noto Sans Mono Light Nerd Font Complete Mono.ttf'
+  font 'Noto Sans Mono Light Nerd Font Complete.ttf'
```
Comment (vitorgalvao) [inline Casks/font-noto-nerd-font-mono.rb]: This (and quite a few others) are not Mono.

Comment (alindeman) [reply Casks/font-noto-nerd-font-mono.rb]: @vitorgalvao Thanks for the catch. I think I've fixed up this mistake.

**Pull request merged**